<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alfalfa_field</class>
 <widget class="QDialog" name="alfalfa_field">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alfalfa field</string>
  </property>
  <property name="windowIcon">
   <iconset resource="game_recources.qrc">
    <normaloff>:/game_backgrounds/pics_project/alfalfa_icon.jpg</normaloff>:/game_backgrounds/pics_project/alfalfa_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/game_backgrounds/pics_project/alfalfa-plant.jpg);</string>
  </property>
  <widget class="QLabel" name="lbl_area">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 255, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="text">
    <string>Area =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_area_value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 255, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_level">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 255, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="text">
    <string>Level =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cultivated_area">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 255, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="text">
    <string>Cultivated area =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_level_value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 255, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cultivated_area_value">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 255, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_upgrade">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/game_backgrounds/pics_project/clipart244133.png);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 0);
color: rgb(60, 0, 255);
border-top-left-radius:20px; 
border-bottom-right-radius:20px;
border-top-right-radius:20px; 
border-bottom-left-radius:20px;</string>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_seed">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/game_backgrounds/pics_project/clipart244133.png);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 0);
color: rgb(60, 0, 255);
border-top-left-radius:20px; 
border-bottom-right-radius:20px;
border-top-right-radius:20px; 
border-bottom-left-radius:20px;</string>
   </property>
   <property name="text">
    <string>Seed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_harvesting">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/game_backgrounds/pics_project/clipart244133.png);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 0);
color: rgb(60, 0, 255);
border-top-left-radius:20px; 
border-bottom-right-radius:20px;
border-top-right-radius:20px; 
border-bottom-left-radius:20px;</string>
   </property>
   <property name="text">
    <string>Harvesting</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="alfalfa_upgrade_pro">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 85, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="prg_harvesting">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 85, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="plow_pro">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 85, 0);
background-image: url(:/game_backgrounds/pics_project/clipart244133.png);</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pro">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/game_backgrounds/pics_project/clipart244133.png);
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 0);
color: rgb(60, 0, 255);
border-top-left-radius:20px; 
border-bottom-right-radius:20px;
border-top-right-radius:20px; 
border-bottom-left-radius:20px;</string>
   </property>
   <property name="text">
    <string>Plow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="build">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>build</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="game_recources.qrc"/>
 </resources>
 <connections/>
</ui>
